### PR TITLE
init: add TODO doc-comment for generated module

### PR DESCRIPTION
**Pull request by MakeNowJust**
Writing a document is good culture 😆

### CODE REVIEW

File: spec/compiler/crystal/tools/init_spec.cr [hunk 1]
```diff
@@ -122,8 +122,9 @@ dependencies:
     describe_file "example/src/example.cr" do |example|
       example.should eq(%{require "./example/*"
 
+# TODO: Write document for Example
```
Comment (Sija) [inline spec/compiler/crystal/tools/init_spec.cr]:
```
# TODO: Write documentation for `Example` class
```

Comment (MakeNowJust) [inline spec/compiler/crystal/tools/init_spec.cr]: But `Example` is not a `class`.

Comment (Sija) [inline spec/compiler/crystal/tools/init_spec.cr]: You're right, it was meant to be `module` of course :)

**Comment by MakeNowJust** [comment]
@mverzilli @RX14 @asterite 🏓

**Pull request merged**